rubocop.yml

```diff
@@ -0,0 +1,15 @@
+Layout:
+  Enabled: false
+
+Style/MultilineIfModifier: # clashes with rubyModifier
+  Enabled: false
+Style/SymbolArray: # clashes with rubyArrayLiteral
+  Enabled: false
+Style/WordArray: # clashes with rubyArrayLiteral
+  Enabled: false
+Style/TrailingCommaInArguments: # clashes with trailingComma
```

Comment:
I think these trailing comma configs don't need to be in here, we should just put into the README that if you have `trailingComma` set then you will need to override these rules.